### DISCUSSION TITLE
Missing useful resources links

### DISCUSSION THREAD

**Discussion by ldynia** [community/community#15997] [Other Feature Feedback, Questions, & Ideas] [answered]
I'd like to see a link to a code search cheat sheet and link to useful prefixes - exactly the same as I see it [here](https://github.com/search/).

### Example:

**ProTip!** Use our [code search cheat sheet](https://github.com/search/advanced), and discover useful [prefixes](https://github.com/search/). To get help with regular expressions, visit [regex101](https://regex101.com/)

**Comment by colinwm** [comment, marked as answer]
Thanks for the feedback! Actually, we do have a link to docs in the search bar, but it's a bit hard to see (click on the ?):

![image](https://user-images.githubusercontent.com/78169714/167425745-94bd50fa-b4e2-47a7-845b-cc57e14d61e6.png)

Our syntax documentation is here: https://cs.github.com/about/syntax, and you can also read our FAQ here: https://cs.github.com/about/faq

I'll see if I can make it a bit more clear how to find this.

**Reply by jsoref** [reply]
It disappears as soon as you enter text into the box.

I don't think there's a good reason for that behavior.

**Comment by ldynia** [comment]
You could follow the solution that you're already using here [https://github.com/search/](https://github.com/search/) Thanks!

# Existing interface
![image](https://user-images.githubusercontent.com/1820831/167484977-e0447359-8847-4a89-8efa-c16665811eeb.png)

# Proposed interface
![image](https://user-images.githubusercontent.com/1820831/167484753-d48aba57-f4e6-4ad0-8f73-36992fce9203.png)

**Reply by ianwilliams1** [reply]
@colinwm , the lack of a "wordy link" to the syntax is very annoying. The live examples really don't give a full picture of the new search capabilities. 

Strongly recommend you either implement the "ProTip" mentioned or add wording:
> You can find the full search [syntax here](https://cs.github.com/about/syntax). Some live examples below:

After all, you provide a link for feedback at the bottom. In fact, given a choice, I'd rather see a link to the documentation instead of the example queries, but both is preferred.

I just gave a quick demo to 400+ team members and completely forgot about the icon being the link to the syntax. 🤦

**Reply by ianwilliams1** [reply]
I should also point out the regular search does NOT have a ❔  icon and it would be really helpful to have that there for consistency, as well as in the other search boxes, such as the Audit Log ( `/settings/audit-log` ) search.
